content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -5,94 +5,46 @@ weight: 40
 ---
 This document describes how to install SODA projects in a local cluster with detailed configuration options. These steps will help you to enable / disable projects. After installation using these steps, you can get the features of all the enabled projects. You can test either through SODA Dashboard UI or CLI
 
-`Hereafter, Hotpot refers to API, Controller and Dock projects`
 
 {{% notice info %}}
-**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
-**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
-{{% /notice %}}
-
-{{% notice info %}}
-**Ubuntu 20.04 support for SODA is added as experimental in Lamu v1.6.0 release through branch `ubuntu2004-experimental`.** <br />
-**If you have a requirement to install SODA on Ubuntu 20.04, please [refer](https://github.com/sodafoundation/installer/tree/ubuntu2004-experimental) or contact us on slack and we will try to help with the setup.**
+**Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04.** <br />
```

Comment:
_Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04_  -> **This installation guide is supported on Ubuntu 18.04 & Ubuntu 20.04.**

Reply:
updated